modules/openfast-library/src/FAST_Library.f90

```diff
@@ -22,8 +22,8 @@ MODULE FAST_Data
    INTEGER(IntKi)                        :: NumTurbines 
    INTEGER,        PARAMETER             :: IntfStrLen  = 1025       ! length of strings through the C interface
    INTEGER(IntKi), PARAMETER             :: MAXOUTPUTS = 4000        ! Maximum number of outputs
-   INTEGER(IntKi), PARAMETER             :: MAXInitINPUTS = 10       ! Maximum number of initialization values from Simulink
-   INTEGER(IntKi), PARAMETER             :: NumFixedInputs = 8
+   INTEGER(IntKi), PARAMETER             :: MAXInitINPUTS = 13       ! Maximum number of initialization values from Simulink
+   INTEGER(IntKi), PARAMETER             :: NumFixedInputs = 11
```

Comment:
With this change--requiring 3 additional fixed inputs to the S-Function--you should also change the example 'OpenLoop.mdl', and `Test01_SIG.mdl` files in `<openfast>/glue-codes/simulink/examples`.

![image](https://user-images.githubusercontent.com/6108781/109180880-97057500-7748-11eb-8a0e-5ee8e2815a42.png)

Reply:
Good catch.  I'll take a look at the sample model.

We will likely be expanding this out to more inputs -- another 40 or the cable control (2 per cable, some models have upwards of 15 already, so some future proofing is needed).

Reply:
I have version 10.2 of Simulink.  Those models were created with version 7.9.  Do you see any issues in updating to the 10.2 version?  Alternatively since the .mdl file is all a form of xml, I could manually port over just the required updates to maintain compatibility with version 7.9

Reply:
I wouldn't worry very much about backward compatibility with Matlab. Version 7.9 was from 2012. People can get a current trial version of Matlab if they want to use these examples.

I would be nice to have an updated picture like this in the documentation. Not sure about the status for porting the old FAST ReadMe with the existing Simulink interface documentation, though.

Reply:
I updated and tested the models on r2020a (on mac).